repo/packages/K/kafka/41/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "packagingVersion": "4.0",
```

Comment:
@elezar @nickbp This doesn't have auto-generated upgradesFrom downgradesTo. Is that expected?

Reply:
The problem is that https://jira.mesosphere.com/browse/INFINITY-2223 was not marked for backporting. I will have a look at what the effort is in doing this.

Reply:
For the record. The backport PR is: mesosphere/dcos-commons#1662